src/devices/Adg731/Metadata/Adg731Metadata.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Device.Spi;
+
+namespace Iot.Device.Adg731
+{
+    /// <summary>
+    /// Metadata class for ADG731
+    /// </summary>
+    public class Adg731Metadata : IDeviceMetadata, ISpiDeviceMetadata, IMultiplexerDeviceMetadata
```

Comment:
please remove the interfaces and create an issue instead for them so that we either add it to all devices or everywhere or nowhere - regardless we should be consistent with what the rest of devices does